websites/migrations/0017_content_subtype.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.1 on 2021-03-29 21:59
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("websites", "0016_remove_site_content_type_constraint"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="websitecontent",
+            name="subtype",
```

Comment:
What is the need for this field ?

Reply:
Given a config that looks like this:

```yml
collections
  - name: navigation
    label: Navigation
    tag: Settings
    files:
      - file: config/menu.sidenav.yml
        label: Side Navigation
        name: sidenav
        fields: [...]
```

If you save a record for side navigation, that record would have "navigation" as the type and "sidenav" as the subtype

Reply:
Why is that field necessary?

Reply:
It's the field that relates the database record to the actual item in the config. In the authoring UI, we need a way to know if there is an existing record for one of those "file" items. In the example above, we would query the db for a record with type="navigation", subtype="sidenav". We could just set "sidenav" as the name, but then we wouldn't be able to query for all objects under "navigation", which is useful to determine which of the "file" items have an object created for them.

Reply:
We could potentially flip that and add a "parent_type" or something like that instead of "subtype". Then that example would have type="sidenav", parent_type="navigation"